doc/TOR.md

```diff
@@ -100,6 +185,42 @@ is necessary to set up a [Tor Hidden Service].
 *NOTE: In the case where only non-persistent addresses are required,
 you don't have to create the hidden service and you can skip this part.*
 
+##### Automatic persistent .onion address
+
+It is possible to generate persistent .onion addresses automatically.
+
+Add the following lines in the `/etc/tor/torrc` file
+(you might already have done this if for example you connected Bitcoin
+over Tor):
+
+````
+ControlPort 9051
+CookieAuthentication 1
+CookieAuthFileGroupReadable 1
+````
+
+Then you can use `--addr=statictor:127.0.0.1:9051` instead of
+`--announce-addr=.onionAddressV3`.
+By default V3 onion addresses are generated.
+
+Note that you have to specify a `--bind-addr` first before using
+`--addr=statictor:`.
+Generally `--bind-addr=127.0.0.1:9735` should work fine.
+
+You can also have multiple announced persistent .onion addresses
+by adding `/torblob=BLOB`, where `BLOB` is up to 64 bytes of
+text.
+You can indicate `--addr=statictor:127.0.0.1:9051/torblob=1
+--addr=statictor:127.0.0.1/torblob=2` to create two different
```

Comment:
at least 32 char should be used to evade collisions

Reply:
Given that you already mix the node privkey and pubkey, is that not enough to evade collisions in practice?  We already assume no two nodes can ever have the same pubkeys.

Reply:
That full magic is only when used without ` /torblob=`  the `torblob `option gives you the opportunity to define in essence your own unique portable tor address with the 32Byte prefix that results after the ED25519-V3 pub key generation by the algo inside tor the human readable onion address part, the second 32byte define the PRF secret, if you define only 32 char the PRF will be from getrandom of sodium. So that the ED25519-V3 algo can  generate unique pub/priv key's.

Reply:
Ah, so you mean that it is not mixed with the ECDH secret?  i.e. the ECDH secret is only used if `/torblob` is not specified?

Reply:
yup that was the intention.

Reply:
in essence an ED25519 key is just some random bits.

Reply:
Okay, will fix.  Unfortunately underlying hardware has to perform regular maintenance shutdown for a while, so perhaps tomorrow.

Reply:
Hope your meat puppet is well and not effected by things i tried to outsource into silicon in the 80ties till then ...